includes/templates/bootstrap/common/tpl_header.php

```diff
@@ -34,7 +34,7 @@
 <?php
 if (!$this_is_home_page) {
 ?>
-                        <li class="nav-item">
+                        <li class="nav-item" title="<?php echo HEADER_TITLE_CATALOG; ?>">
```

Comment:
I'm thinking that a better approach might be to use an `aria-label="{text}"` on the associated nav-link anchor tags.

Reply:
But that does not pop-up on hover, as far as I can see.
I put them on the li so the hover trigger-area is as wide as possible.

Reply:
While using icons instead of the as-shipped version:

![image](https://github.com/lat9/ZCA-Bootstrap-Template/assets/2685585/fe48b215-9738-4e02-b2f6-4fe4c279e213)

... your sighted customers visual 'clue' that the icon is clickable (plus the cursor changes to "hand" instead of "pointer":

![image](https://github.com/lat9/ZCA-Bootstrap-Template/assets/2685585/31ece38f-84a1-4cd8-b774-051a6a066b29)

Remember, too, that 'hover' only applies to desktop viewers.

Reply:
I don't see any difference between those images.

I get the hover and popup at all viewport sizes.

Reply:
The difference (albeit subtle) is the hover-color (slightly greyed) and no underline.  If you're viewing via a browser's mobile rendering and you're not running with a display that supports touch, you will see the title on hover. Try viewing the page on a "real" phone/tablet.

Reply:
Ah, since I have no text...no underlines.
And to reduce wasted space I don't show "unecessary" icons:
when in the shop, but no cart: don't show the cart or checkout
when in the shop, with cart: show the cart but not checkout
when in the cart: show checkout but not the cart
when in checkout: show cart but not checkout

And since we are long way from live, I've not got in the habit with checking with a real phone!